insert-movies-to-DB.ts

```diff
@@ -1,36 +1,15 @@
 import fs from "fs";
 
 import { Movie } from "./model/Movie";
+import { MovieParams } from "./util/types";
 
 export async function insertMoviesData(): Promise<void> {
   const movies = JSON.parse(fs.readFileSync("./movies.json", "utf8"));
 
-  const promises = movies.map(async (movie: any) => {
-    const newMovie = new Movie();
-    newMovie.Title = movie.Title;
-    newMovie.Year = movie.Year;
-    newMovie.Rated = movie.Rated;
-    newMovie.Released = movie.Released;
-    newMovie.Runtime = movie.Runtime;
-    newMovie.Genre = movie.Genre;
-    newMovie.Director = movie.Director;
-    newMovie.Writer = movie.Writer;
-    newMovie.Actors = movie.Actors;
-    newMovie.Plot = movie.Plot;
-    newMovie.Language = movie.Language;
-    newMovie.Country = movie.Country;
-    newMovie.Awards = movie.Awards;
-    newMovie.Poster = movie.Poster;
-    newMovie.Metascore = movie.Metascore;
-    newMovie.imdbRating = movie.imdbRating;
-    newMovie.imdbVotes = movie.imdbVotes;
-    newMovie.imdbID = movie.imdbID;
-    newMovie.Type = movie.Type;
-    newMovie.Response = movie.Response;
-    newMovie.Images = movie.Images;
-    newMovie.ComingSoon = movie.ComingSoon || false;
-
-    await newMovie.save();
+  const promises = movies.map(async (movie: MovieParams) => {
+    let newMovie = new Movie();
+    Object.assign(newMovie, movie);
+    newMovie = await newMovie.save();
```

Comment:
You can just have `return new Movie(movie).save()`.
Also, as in improvement, no need for separate constant promises when you can just have 
```javascript
await Promise.all(movies.map(async movie => (
...
)))
```

Reply:
Changed. I added constructor in Movie model, so I could send the movie as you said above.

Reply:
Sorry, I didn't write it correct, the movie should be sent in the save method (or however it is defined in the Movie class).

Reply:
It works this way as well, can you see if it is okay or if I need to change it again :D

Reply:
I am not able to see the change, maybe it is not pushed to the branch? However, it is better to send the whole movie object as part of the method instead of the constructor. The reason for this is, if we have more methods in the same class, we do not want to be forced to send the movie object with all of its properties, when in fact maybe we would just need to have the id.